R/relevel.R

```diff
@@ -1,72 +1,30 @@
-#' Reorganize a nested parse table
+#' Flatten some token in the nested parse table
 #'
 #' Certain tokens are not placed optimally in the nested parse data with
 #'   [compute_parse_data_nested()]. For example, the token of arithmetic
 #'   operations 1 + 1 + 1 should all be on the same level of nesting since
 #'   the indention is the same for all but the first two terminals. Setting the
 #'   indention correcly is easier to achieve if they are put on the same level
 #'   of nesting.
-#' @examples
-#' code <- "1+1+1+1"
-#' # tree-structure with compute_parse_data_nested
-#' styler:::create_tree(code, re_nest = FALSE)
-#' # tree-structure with re_nest post-processing
-#' styler:::create_tree(code, re_nest = TRUE)
-#' @inheritParams move_children_up
-re_nest <- function(pd_nested) {
-  if (is.null(pd_nested)) return()
-  pd_nested <- pd_nested %>%
-    move_children_up(token = c("'+'", "'-'", "SPECIAL", "'/'", "'*'")) %>%
-    mutate(child = map(child, re_nest))
-  pd_nested
+flatten_partially <- function(pd_nested) {
```

Comment:
Currently we flatten only operators, right? Perhaps the function name could reflect this.